### DISCUSSION TITLE
t.requestAdapterInfo is not a function

### DISCUSSION THREAD

**Discussion by Fabioni** [visgl/luma.gl#2303] [General]
On https://luma.gl/examples/showcase/instancing/ clicking "WebGPU" shows 
> t.requestAdapterInfo is not a function
and does not show any canvas. The WebGL variant works as expected.

What does it mean that the example website shows this error message?

**Comment by ibgreen** [comment]
@fabioni, thanks for reporting!

The WebGPU `requestAdapterInfo()` function was removed in in [Chrome 131](https://developer.chrome.com/blog/new-in-webgpu-131). It seems that since WebGPU API is still quite new, Chrome developers feel they can make breaking changes like this and expect users to quickly fix their code.

We have already fixed this issue on the v9 branches (https://github.com/visgl/luma.gl/issues/2292), so we should redeploy the website to include the change. However there are some in other in-progress changes that need to go in as well. Will try to take another look soon.
